packages/world-modules/src/modules/erc20-puppet/ERC20Module.sol

```diff
@@ -68,6 +72,7 @@ contract ERC20Module is Module {
 
     // Register the ERC20 in the ERC20Registry
     if (!ResourceIds.getExists(ERC20_REGISTRY_TABLE_ID)) {
+      world.registerNamespace(MODULE_NAMESPACE_ID);
```

Comment:
what if the namespace exists but the table doesn't? should we put this in a separate conditional?

Reply:
since both are registered in these two lines, and the table could not be registered if the namespace is not registered, i think it's fine to keep both in one conditional